validators/RequestSchema.js

```diff
@@ -145,6 +145,35 @@ const DeleteMediaFileRequestSchema = Joi.object().keys({
   sha: Joi.string().required(),
 })
 
+const UpdateNavigationRequestSchema = Joi.object().keys({
+  content: Joi.object()
+    .keys({
+      logo: Joi.string().allow(""),
+      links: Joi.array()
+        .items(
+          Joi.object()
+            .keys({
+              title: Joi.string().required(),
+              url: Joi.string(),
+              collection: Joi.string(),
+              resource_room: Joi.boolean().valid(true),
```

Comment:
just a question, why is `false` not allowed

Reply:
A navbar item will only contain one of the following set of params:

- Url
- Collection
- Resource_room
- Url + sublinks

This means that if we see `resource_room` as the params, it should always be `true`, otherwise the param will just not exist